sirepo/auth/__init__.py

```diff
@@ -82,11 +82,11 @@
 _cfg = None
 
 
-def init_quest(qcall):
+def init_quest(qcall, top_level_call_api=False):
```

Comment:
I think it is worth documenting this method. Not exactly clear to me what top_level_call_api is and why this init_quest needs to be different from all of the others.

Reply:
I documented to show that it was unpublished 695c9ef3f83ac271fecd1f4f45f84129ddb26239